packages/polymarket-notifier/src/polymarketNotifier.js

```diff
@@ -0,0 +1,223 @@
+// This notifier is used to monitor and compare Optimistic Oracle contract proposals against the Polymarket API endpoint.
+
+const { Datastore } = require("@google-cloud/datastore");
+const datastore = new Datastore();
+const abi = require("./abi/abi");
+const Web3 = require("web3");
+const ethers = require("ethers");
+const uma = require("@uma/sdk");
+const { getAddress } = require("@uma/contracts-node");
+
+const web3 = new Web3(process.env.CUSTOM_NODE_URL);
+const provider = new ethers.providers.WebSocketProvider(process.env.CUSTOM_NODE_URL);
+
+const polymarketContract = "0xCB1822859cEF82Cd2Eb4E6276C7916e692995130";
```

Comment:
this should be stored in the networks file for the associated network. see: https://github.com/UMAprotocol/protocol/blob/master/packages/core/networks/137.json